build.gradle

```diff
@@ -35,11 +35,11 @@ subprojects {
     afterEvaluate { project ->
         if (project.hasProperty("android")) {
             android {
-                compileSdkVersion 32
+                compileSdkVersion 33
 
                 defaultConfig {
                     minSdkVersion 21
-                    targetSdkVersion 32
+                    targetSdkVersion 33
```

Comment:
Do we have enough Android 13 devices to test on? My Pixel 3a doesn't run Android 13. Latest is Android 12. Maybe I need a new device to develop on?

Reply:
you could use Android 13 emulator for now. Should be enough.